outmon/vertexmon.go

```diff
@@ -190,11 +192,26 @@ func (vm *vertexMonitor) printError() bool {
 	switch {
 	case reErrExitCode.MatchString(errString):
 		m := reErrExitCode.FindStringSubmatch(errString)
-		errString = fmt.Sprintf(""+
-			"      The%s command\n"+
-			"          %s\n"+
-			"      did not complete successfully. Exit code %s",
-			internalStr, indentOp, m[2])
+
+		// Ignore the Error, default case will print it as a string using the source, so we won't miss any data.
+		exitCode, _ := strconv.ParseUint(m[2], 10, 32)
+
+		switch exitCode {
+		case 137:
+		case math.MaxUint32:
+			errString = fmt.Sprintf(""+
+				"      The%s command\n"+
+				"          %s\n"+
+				"      was terminated because the satellite ran out of memory.",
```

Comment:
I had it that way originally; the thought was this probably wont be used locally? It could though.

@brandonSc @vladaionescu Y'all had opinions on wording. Im not worried about it; what is the consensus here?

Reply:
`was terminated because of an out of memory error. If you are using a satellite or other remote buildkit, it is the remote system that ran out of memory.`

Reply:
Updated message, if you are good with it @alexcb let me know and I will merge

Reply:
looks good!